rotkehlchen/chain/scroll/constants.py

```diff
@@ -0,0 +1,23 @@
+from rotkehlchen.chain.evm.types import NodeName, WeightedNode, string_to_evm_address
+from rotkehlchen.constants import ONE
+from rotkehlchen.fval import FVal
+from rotkehlchen.types import SupportedBlockchain, Timestamp, deserialize_evm_tx_hash
+
+SCROLL_ETHERSCAN_NODE_NAME = 'scroll etherscan'
+SCROLL_GENESIS = Timestamp(1696917600)
+SCROLL_ETHERSCAN_NODE = WeightedNode(
+    node_info=NodeName(
+        name=SCROLL_ETHERSCAN_NODE_NAME,
+        endpoint='',
+        owned=False,
+        blockchain=SupportedBlockchain.SCROLL,
+    ),
+    weight=ONE,
+    active=True,
+)
+
+ARCHIVE_NODE_CHECK_ADDRESS = string_to_evm_address('0xCace5b3c29211740E595850E80478416eE77cA21')
```

Comment:
Is there a definition for when a node is pruned or archived? I mean for Scroll, since the genesis block was last year, I think all nodes are including the data for old address and transaction. But that might change in time...

Reply:
yes these are standard options in how you run a node. There is no way to know it apart from querying the node. And even if the chain is 1 day old most nude would not be archive and maybe prune by default. This is to save space.

Reply:
addressed in https://github.com/rotki/rotki/pull/7584/commits/a72c9e4587bf541c44eb6f57f50aed3250a07ae2